src/commands/quick-start/default-quick-start.ts

```diff
@@ -261,13 +278,85 @@ export class DefaultQuickStartCommand extends BaseCommand implements QuickStartC
+              await this.localConfig.load();
+              await this.remoteConfig.loadAndValidate(argv);
+
+              const self = this;
+              const subTasks: SoloListrTask<QuickStartSingleDeployContext>[] = [];
+
+              const accountsToCreate = [
+                ...predefinedEcdsaAccounts,
+                ...predefinedEcdsaAccountsWithAlias,
+                ...predefinedEd25519Accounts,
+              ];
+
+              await self.accountManager.loadNodeClient(
+                config.namespace,
+                self.remoteConfig.getClusterRefs(),
+                context_.config.deployment,
+              );
+
+              for (const [index, account] of accountsToCreate.entries()) {
+                // inject index to avoid closure issues
+                ((index: number, account: PredefinedAccount) => {
+                  subTasks.push({
+                    title: `Creating Account ${index}`,
+                    task: async (
+                      context_: QuickStartSingleDeployContext,
+                      subTask: SoloListrTaskWrapper<QuickStartSingleDeployContext>,
+                    ): Promise<void> => {
+                      await helpers.sleep(Duration.ofMillis(100 * index));
+
+                      const createdAccount = await self.accountManager.createNewAccount(
+                        context_.config.namespace,
+                        account.privateKey,
+                        account.balance.to(HbarUnit.Hbar).toNumber(),
+                        account.alias,
+                        context_.config.context,
+                      );
+
+                      context_.createdAccounts.push({
+                        accountId: AccountId.fromString(createdAccount.accountId),
+                        data: account,
+                        alias: createdAccount.accountAlias,
+                      });
+
+                      subTask.title = `Account created: ${createdAccount.accountId.toString()}`;
+                    },
+                  });
+                })(index, account);
+              }
+
+              // set up the sub-tasks
+              return task.newListr(subTasks, {
+                concurrent: true,
+                rendererOptions: {
+                  collapseSubtasks: false,
+                },
+              });
+            },
+          },
           {
             title: 'Finish',
             task: async (context_: QuickStartSingleDeployContext): Promise<void> => {
               this.showQuickStartUserNotes(context_);
               this.showVersions();
               this.showPortForwards();
 
+              if (context_.createdAccounts) {
+                context_.createdAccounts.sort((a: CreatedPredefinedAccount, b: CreatedPredefinedAccount): number =>
+                  a.accountId.compare(b.accountId),
+                );
+                this.showCreatedAccounts(context_.createdAccounts);
+              }
```

Comment:
```suggestion
              this.showCreatedAccounts(context_.createdAccounts);
```
move the if check and the sorting inside of showCreatedAccounts()